src/OpenTelemetry.Instrumentation.SqlClient/Implementation/SqlActivitySourceHelper.cs

```diff
@@ -0,0 +1,37 @@
+// <copyright file="SqlActivitySourceHelper.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+using System;
+using System.Diagnostics;
+
+namespace OpenTelemetry.Instrumentation.SqlClient.Implementation
+{
+    /// <summary>
+    /// Helper class to hold common properties used by both SqlClientDiagnosticListener on .NET Core
+    /// and SqlEventSourceListener on .NET Framework.
+    /// </summary>
+    internal class SqlActivitySourceHelper
+    {
+        public const string ActivitySourceName = "OpenTelemetry.SqlClient";
+        public const string ActivityName = ActivitySourceName + ".Execute";
+
+        public const string MicrosoftSqlServerDatabaseSystemName = "mssql";
```

Comment:
nit: Might be better to move this definition to semantic conventions, that's where it came from (https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/database.md#connection-level-attributes).

Reply:
Thanks, I'll update this! Other things ("db.statement") are being pulled from semantic conventions so better to be consistent.